edx_django_utils/ip/internal/ip.py

```diff
@@ -357,15 +357,16 @@ def _get_client_ips_via_trusted_header(request):
         if closest_client_ip := _get_trusted_header_ip(request, header_name, index):
             # The equality check in this predicate is why we use parsed IP
             # addresses -- ::1 should compare as equal to 0:0:0:0:0:0:0:1.
-            external_chain = _remove_tail(full_chain, lambda ip: ip != closest_client_ip)  # pylint: disable=cell-var-from-loop
+            # pylint: disable=cell-var-from-loop
+            external_chain = _remove_tail(full_chain, lambda ip: ip != closest_client_ip)
```

Comment:
Nit: Why this change? Did it also have line too long? You could disable that as well. You can leave this, but I like getting this stuff off to the side.

Reply:
Yeah, pycodestyle E501 line-too-long, and I was having trouble figuring out how to disable _just_ that warning (rather than using `noqa`). I can take another look to see if there's a way to do that,

Reply:
OK, apparently `noqa` has to appear _first_ in the comment, and doesn't accept things like `E501` -- it just entirely disables all errors of any sort on that line.

Reply:
Are you saying the recommendation to use `# noqa: E501, pylint: disable=line-too-long` doesn't work?

Reply:
I’m just curious, but feel free to leave this on two lines. I don’t mean to get us lost in this. Also why I wish there was a doc for this, if it could be solved once.